project/WORKSPACE

```diff
@@ -54,3 +54,76 @@ http_jar (
     name = "apache-commons-text",
     url = "https://repo1.maven.org/maven2/org/apache/commons/commons-text/1.9/commons-text-1.9.jar"
 )
+
+http_archive(
+    name = "com_google_j2cl",
+    strip_prefix = "j2cl-master",
+    url = "https://github.com/google/j2cl/archive/master.zip",
+)
+
+load("@com_google_j2cl//build_defs:repository.bzl", "load_j2cl_repo_deps")
+load_j2cl_repo_deps()
+
+load("@com_google_j2cl//build_defs:rules.bzl", "setup_j2cl_workspace", "j2cl_maven_import_external")
+setup_j2cl_workspace()
+
+http_archive(
+    name = "com_google_elemental2",
+    strip_prefix = "elemental2-master",
+    url = "https://github.com/google/elemental2/archive/master.zip",
```

Comment:
ditto

Reply:
I have checked [version 1.1.0](https://github.com/google/elemental2/releases) and it works, let's use that one.